apps/src/netsim/NetSimRouterLogModal.html.ejs

```diff
@@ -29,7 +29,11 @@ function sortMarkupFor(sortKey) {
 <div class="modal-dialog modal-lg">
     <div class="modal-header">
       <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-      <h4 class="modal-title"><%= i18n.logBrowserHeader() %></h4>
+      <% if (isAllRouterMode) { %>
```

Comment:
Just at the head of file. I don't think we have a formal convention for this so use your best judgement; I think of these parameters as the moral equivalent of method parameters and its helpful for the reader to know what they are.

 (There must be a way to do an ".ejs" comment vs. something that would appear in the markup; that would be preferable.)